tools/cldr-apps/src/main/webapp/js/esm/cldrStatus.js

```diff
@@ -396,7 +396,23 @@ function isDashboard() {
   return getCurrentSpecial() === "r_vetting_json";
 }
 
+/**
+ * Is auto-import of old votes in progress? (Boolean)
+ *
+ * Used for inhibiting $('#left-sidebar').hover (Locale chooser)
+ */
+let isAutoImportBusy = false;
+
+function getAutoImportBusy() {
+  return isAutoImportBusy;
+}
+
+function setAutoImportBusy(busy) {
+  isAutoImportBusy = !!busy;
```

Comment:
nice

Reply:
another way to do make sure the result is boolean